### ISSUE TITLE
Workbox bundle assets

### ISSUE THREAD

**Issue by michaeldanielbell** [Rainrider/rollup-plugin-workbox-build#11]
Hi, 

Is this plugin supposed to cache the built assets?

I had to add this to get it working

```
writeBundle(bundle) {
      options.importScripts = Object.keys(bundle);
      return build(options)
        .then(doRender)
        .catch(console.error);
    },
```

I'm using rollup 1.15.4

**Comment by Rainrider** [comment]
I don't think that using `Object.keys(bundle)` without further modification is the right general solution in this case. The file paths are relative to the rollup output dir and depend on the underlying OS, so they may break depending on where you generate the service worker.

Do you really intend `importScript`ing the whole bundle though? If you are just looking to populate worbox' precache manifest, then using [`globDirectory`](https://developers.google.com/web/tools/workbox/modules/workbox-build#generateSW-globDirectory) may be a better way.
